src/js/components/account/change-password.js

```diff
@@ -0,0 +1,90 @@
+import { API } from "../../api";
+import {
+  getDiv,
+  getForm,
+  getInput,
+  getLabel,
+  getSubmitButton,
+} from "../../utils/element-factory";
+
+export class ChangePassword {
+  constructor() {
+    const $successTemplate = $(".styles .form-styles .w-form-done").clone();
+    const $failTemplate = $(".styles .form-styles .w-form-fail").clone();
+
+    const defaultBaseUrl = "https://muni-portal-backend.openup.org.za";
+    const endPoint = "/api/accounts/change-password/";
+
+    const $changeFormContainer = getDiv("form w-form");
+    const $form = getForm(`${defaultBaseUrl}${endPoint}`, "post");
+    const $submitButton = getSubmitButton("Change Password");
+
+    const $formElementsContainer = $("<div />");
+    $formElementsContainer.append(getLabel("current password"));
+    $formElementsContainer.append(getInput("password", "old_password"));
+
+    $formElementsContainer.append(getLabel("new password"));
+    $formElementsContainer.append(getInput("password", "password"));
+
+    $formElementsContainer.append(getLabel("confirm new password"));
```

Comment:
For this and the previous two instances I believe we are using sentence case for labels - @jbothma, am I correct?

Reply:
Thanks @schalkneethling - I've updated the labels to sentence case